pkg/arvo/app/chat-cli.hoon

```diff
@@ -363,13 +365,20 @@
         ;~  (glue ace)
           (tag %create)
           security
-          ;~(plug path (punt ;~(pfix ace glyph)))
+          path
+          (punt ;~(pfix ace glyph))
+          (punt ;~(pfix ace ;~(pose (cold %| bar) (cold %& pad))))
```

Comment:
Pretty sure this won't parse correctly, demanding at least two spaces after the `path` (you're punting, but the `glue`'s still there!). You want:

```hoon
          ;~  plug
            path
            (punt ;~(pfix ace glyph))
            (punt ;~(pfix ace ;~(pose (cold %| (jest 'n')) (cold %& (jest 'y')))))
          ==
```

Untested, but you get the idea. Note also the `y` and `n` instead of `& |` to avoid ambiguity with the glyph parsing. (Talk had a parser-primitive for that, should probably bring it back.)

Reply:
I think you want ++fuss from hoon.hoon

Reply:
done, thanks!

Reply:
This uses `+fuss` now, but the code below doesn't, and both still want too many spaces.

Reply:
I've tested out the ;create parser, and it only wants a single space in between each option. Currently doing ;join

Reply:
Oh huh, sorry, I think gh was showing me outdated code. (Or I was interpreting it as the old code.) This does indeed seem good. Carry on! (: